lib/util/hud_utility.rb

```diff
@@ -270,166 +59,23 @@ def homeless_project_type_numbers
     ].freeze
   end
 
-  # 2.02.C
-  def tracking_method(id, reverse = false)
-    map = tracking_methods
-
-    _translate map, id, reverse
-  end
-
+  # Override
   def tracking_methods
     # FIXME Do we want the nil mapping?
-    ::HudLists.tracking_method_map.merge(nil => 'Entry/Exit Date')
-  end
-
-  # 2.06.1
-  def funding_sources
-    ::HudLists.funding_source_map
-  end
-
-  def funding_source(id, reverse = false)
-    map = funding_sources
-    _translate map, id, reverse
-  end
-
-  # 2.07.4
-  def household_type(id, reverse = false)
-    map = household_types
-
-    _translate map, id, reverse
-  end
-
-  def household_types
-    ::HudLists.household_type_map
-  end
-
-  # 2.07.5
-  def bed_type(id, reverse = false)
-    map = bed_types
-
-    _translate map, id, reverse
-  end
-
-  def bed_types
-    ::HudLists.bed_type_map
-  end
-
-  # 2.07.6
-  def availability(id, reverse = false)
-    map = availabilities
-
-    _translate map, id, reverse
-  end
-
-  def availabilities
-    ::HudLists.availability_map
-  end
-
-  # 2.7.B
-  def youth_age_group(id, reverse = false)
-    map = youth_age_groups
-
-    _translate map, id, reverse
-  end
-
-  def youth_age_groups
-    ::HudLists.youth_age_group_map
-  end
-
-  # 2.03.4
-  def geography_type(id, reverse = false)
-    map = geography_types
-
-    _translate map, id, reverse
-  end
-
-  def geography_types
-    ::HudLists.geography_type_map
-  end
-
-  # 2.02.D
-  def housing_type(id, reverse = false)
-    map = housing_types
-
-    _translate map, id, reverse
-  end
-
-  def housing_types
-    ::HudLists.housing_type_map
-  end
-
-  # 2.02.8
-  def target_population(id, reverse = false)
-    map = target_populations
-
-    _translate map, id, reverse
-  end
-
-  def target_populations
-    ::HudLists.target_population_map
-  end
-
-  # 2.02.9
-  def h_o_p_w_a_med_assisted_living_fac(id, reverse = false)
-    map = h_o_p_w_a_med_assisted_living_facs
-
-    _translate map, id, reverse
-  end
-
-  def h_o_p_w_a_med_assisted_living_facs
-    ::HudLists.hopwa_med_assisted_living_fac_map
-  end
-
-  # 3.01.5
-  def name_data_quality(id, reverse = false)
-    _translate(name_data_quality_options, id, reverse)
-  end
-
-  def name_data_quality_options
-    ::HudLists.name_data_quality_map
-  end
-
-  # 3.02.2
-  def ssn_data_quality(id, reverse = false)
-    _translate(ssn_data_quality_options, id, reverse)
-  end
-
-  def ssn_data_quality_options
-    ::HudLists.ssn_data_quality_map
-  end
-
-  # 3.03.2
-  def dob_data_quality(id, reverse = false)
-    map = dob_data_quality_options
-
-    _translate map, id, reverse
-  end
-
-  def dob_data_quality_options
-    ::HudLists.dob_data_quality_map
-  end
-
-  # 3.05.1
-  def ethnicity(id, reverse = false)
-    map = ethnicities
-
-    _translate map, id, reverse
+    {
+      nil => 'Entry/Exit Date',
+      0 => 'Entry/Exit Date',
+      3 => 'Night-by-Night',
+    }.freeze
```

Comment:
Is this really needed?

Reply:
Looks like we are still using it, but only on the Project edit page, so we could just fix that up.
`app/views/projects/edit.haml`

Reply:
https://github.com/greenriver/hmis-warehouse/pull/3316/commits/f2a011e8c17967e4239605663455ccd1cdf26e5d